src/gql/generated/graphcache.ts

```diff
@@ -1,6 +1,6 @@
 /* eslint-disable no-use-before-define */
 import { cacheExchange } from '@urql/exchange-graphcache'
-import type {
```

Comment:
The import statement here should not be changed. Just revert the type removal when you regenerate the `gql` files.